tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -30,6 +36,73 @@ class FileConverterTests
         private string _testProjectFilePath;
         private string _testAreaFullPath;
 
+        private ProjectAnalyzer _webFormsProjectAnalyzer;
+        private WorkspaceManagerService _blazorWorkspaceManager;
+        
+        [SetUpFixture]
+        public class WebForms2BlazorTestFixture : AwsRulesBaseTest
```

Comment:
I would try to give this setup fixture a more descriptive name

Reply:
That was a name I defaulted to. I agree, it can use a more descriptive name such as "DownloadTestProjectsFixture" or something. 

Also, this fixture and related fields should go in its own file.